airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -246,14 +245,14 @@ abstract class BasicFunctionalityIntegrationTest(
                     minimumGenerationId = 0,
                     syncId = 42,
                 )
+            val stream = makeStream("test_stream")
```

Comment:
nit: I think this is the only call to makeStream now, so we might as well just inline it

(the function was just for convenience when there were actually two streams, but now there's only 1 :shrug: )

Reply:
addressed in https://github.com/airbytehq/airbyte/pull/48583/commits/a2f1f7409d19ff9b618ae9e35dc883ee7749a146